brands/amenity/bank.json

```diff
@@ -4622,10 +4622,12 @@
     }
   },
   "amenity/bank|Moldindconbank": {
-    "locationSet": {"include": ["001"]},
+    "locationSet": {"include": ["md"]},
     "tags": {
       "amenity": "bank",
       "brand": "Moldindconbank",
+      "brand:wikidata": "Q46730",
```

Comment:
This wikidata doesn't look like it's the right one

Reply:
Good catch
I